### DISCUSSION TITLE
Automatically copy the values from other fields

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#16514] [How to]
Hi,

I created a new module named daily task and i want to add contact as one of the field. 

![image](https://user-images.githubusercontent.com/96602176/191928566-386cd6a5-4949-4603-aa9d-91c93e808327.png)

Whenever i add a contact from the related module, automatically i want to update the mobile number and email fields. 

I tried to use update field and update related fields in the workflow automation. but unable to achieve my goal. 

Could you please guide me to update the fields .

Thanks & regards,
Rajasekaran

**Comment by apcloic** [comment]
@RajasekaranJeni 
Did you configure the mapping between contact module and your module ?
https://gitstable.yetiforce.com/index.php?module=MappedFields&parent=Settings&view=List

**Reply by Ilamparithi-Tamizhanangu** [reply]
Hi,

I tried to map the field names in the gitstable demo version. 

i followed the following procedure.

1. Created the new text field "TAX Number" in the opportunity field. 
2. Go to field mapping and add module as opportunity  and related module as accounts.
3. Link the tax number field from both the modules 
4. no conditions specified. I want to map for all the opportunities
5. All the roles are added. and saved the mapping. 
6. In the new opportunity record, after adding the account name, TAX number was not updated automatically. 
7. When i try to add new field in the opportunity module as "Values from related module", Accounts module was not available in the related module list. 
![image](https://user-images.githubusercontent.com/96602176/192006532-25c114d5-7635-42db-9df1-21d50dd3167e.png)

8. I tried to map it in both the direction (Module : Accounts + Related module : Opportunity and Module : Opportunity and Related module as Accounts)

Could you please guide me to solve this. 

Thankinh you.

**Reply by apcloic** [reply]
Sorry, you're right as mapped is for generate record button
Check [this file](https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/app_data/moduleHierarchy.php)
#14123
#15680
